connectors/yarpc/yarpc.go

```diff
@@ -314,7 +314,24 @@ func (c *Connector) Remove(ctx context.Context, ei *dosa.EntityInfo, keys map[st
 		return errors.Wrap(err, "YARPC Remove failed")
 	}
 	return nil
+}
+
+// RemoveRange removes all entities within the range specified by the columnConditions.
+func (c *Connector) RemoveRange(ctx context.Context, ei *dosa.EntityInfo, columnConditions map[string][]*dosa.Condition) error {
+	conditions, err := createRpcConditions(columnConditions)
+	if err != nil {
+		errors.Wrap(err, "RemoveRange failed")
```

Comment:
ugh. how did I miss this. This is just silly. Same with the one below. I'm going to write tests to exercise these.